### PR TITLE
Updates index.mako

### PR DESCRIPTION
**Pull request by sumit4613**
I guess the latest versions in the documentation shall be updated to their latest stable versions.

### CODE REVIEW

File: index.mako [hunk 1]
```diff
@@ -113,13 +113,13 @@ pre-commit --version
 ```yaml
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v2.4.0
```
Comment (asottile) [inline index.mako:116]: this isn't the latest version

**Comment by sumit4613** [comment]
@asottile i didn't knew that these versions were generated, thanks for letting me know. 🙂

**Comment by asottile** [comment]
they're not yet -- but it would be better if they were instead of many manual PRs

**Comment by sumit4613** [comment]
You're right, I'll try to implement this.

**Pull request closed without merging**